gemfirexd/client/src/main/java/com/pivotal/gemfirexd/internal/client/net/NetConnection.java

```diff
@@ -2360,6 +2360,7 @@ else if (agent != null) {
               final String password = getDeferredResetPassword();
               flowConnect(password, this.securityMechanism_, false);
 
+              setCurrentSchema();
```

Comment:
Is this call required, since lower down setCurrentSchema() is being called in any case?